trueos-handbook/install.rst

```diff
@@ -9,6 +9,290 @@ graphical desktop directly onto a hard drive or into a virtual machine
 using virtualization software such as
 `VirtualBox <https://www.virtualbox.org/>`_.
 
+.. index:: Download and Prepare to Install
+.. _Download and Prepare to Install:
+
+Download and Prepare to Install
+===============================
+
+|trueos| uses a rolling release model rather than versioned releases.
+There are two primary options of |trueos| install: STABLE and UNSTABLE:
+
+1. STABLE is meant to be less frequently updated and synchronized with
+   FreeBSD. This means users see less experimental work and generally
+   have a smoother experience. However, users on STABLE also typically
+   wait longer for bugfixes and patches to be available. While some
+   |trueos| development may be backported to STABLE early, FreeBSD
+   patches and port synchronization is done on a six-month schedule.
+
+2. UNSTABLE is the full leading edge of TrueOS and FreeBSD development.
+   Patches are very frequent, but can incorporate experimental work from
+   both the |trueos| and FreeBSD projects. UNSTABLE is recommended for
+   users who need the absolute latest work from FreeBSD or |trueos| and
+   are willing to tolerate breakage or less system stability. It is also
+   recommended for users who want to test and contribute patches to
+   FreeBSD or |trueos|.
+
+Periodically, the |sysadm| :sysclbk:`Update Manager <update-manager>`
+provides patches to update the operating system. By default, users who
+install STABLE receive updates from the STABLE track, and UNSTABLE users
+are on the UNSTABLE track. It is possible to switch update tracks
+post-installation. See the :ref:`Updating TrueOS` section for
+instructions on switching update repositories.
+
+Installation files can be downloaded from the
+`TrueOS® website <https://www.trueos.org/downloads/>`_ or the
+`PC-BSD® CDN <http://iso.cdn.pcbsd.org/>`_.
+
+Several types of files are available for download. Any file with the
+naming convention *TrueOS-<year>-<month>-<day>-x64-[DVD/USB].[iso/img]*
+are STABLE or UNSTABLE installation files, depending on the download
+directory:
+
+* `STABLE <http://download.trueos.org/master/amd64/>`_: Files in the
+  :file:`master/amd64` directory are the STABLE version of |trueos|.
+
+* `UNSTABLE <http://download.trueos.org/unstable/amd64/>`_: Files in the
+  :file:`unstable/amd64` directory are the UNSTABLE version of |trueos|.
+
```

Comment:
The download page did get reworked, so double/triple check that these links are pointed to the right spot.